ambry-clustermap/src/main/java/com/github/ambry/clustermap/DatacenterInitializer.java

```diff
@@ -109,15 +109,10 @@ public void start() {
     Utils.newThread(() -> {
       try {
         DcInfo dcInfo;
-        switch (dcZkInfo.getReplicaType()) {
-          case DISK_BACKED:
-            dcInfo = initializeHelixDatacenter();
-            break;
-          case CLOUD_BACKED:
-            dcInfo = initializeCloudDatacenter();
-            break;
-          default:
-            throw new UnsupportedOperationException("Unknown replica type: " + dcZkInfo.getReplicaType());
+        if (dcZkInfo.getReplicaType() == ReplicaType.DISK_BACKED) {
+          dcInfo = initializeHelixDatacenter();
+        } else {
+          throw new UnsupportedOperationException("Unknown replica type: " + dcZkInfo.getReplicaType());
```

Comment:
So the next PR would be remove this ReplicaType, right?

Reply:
ReplicaType.CLOUD_BACKED is needed for recovery flow.